src/inspect_ai/_cli/main.py

```diff
@@ -53,7 +53,7 @@ def inspect(ctx: click.Context, version: bool) -> None:
 def main() -> None:
     set_exception_hook()
     init_dotenv()
-    inspect(auto_envvar_prefix="INSPECT")
+    inspect(auto_envvar_prefix="INSPECT")  # pylint: disable=no-value-for-parameter
```

Comment:
I didn't dive into way this one was triggering. I assume it's because of the `@click` stuff, but I'm not sure.

Reply:
Yeah, the `@click` decorator injects those so it should be fine.